doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -5,37 +5,37 @@ Introduction
 ------------
 
 The Anuket Kubernetes Reference Architecture (RA) is intended to be an industry
-standard independent Kubernetes reference architecture that is not tied to any
-specific offering or distribution. No vendor-specific enhancements are required
-in order to achieve conformance to the principles of Anuket specifications; conformance is achieved by
-using upstream components or features that are developed by the open source
-community. This allows operators to have a common Kubernetes-based architecture
-that supports any conformant VNF or CNF deployed on it to operate as expected.
-The purpose of this chapter is to outline all the components required to provide
-Kubernetes in a consistent and reliable way. The specification of how to use
-these components is detailed in Chapter 04 :ref:`chapters/chapter04:component level architecture`.
-
-Kubernetes is already a well documented and widely deployed Open Source project
+standard-independent Kubernetes reference architecture that is not tied to any
+specific offering or distribution. No vendor-specific enhancements are required,
+in order to achieve conformance to the principles of Anuket specifications.
+Conformance is achieved by using upstream components or features that are
+developed by the open source community. This allows operators to have a common
+Kubernetes-based architecture that supports any conformant VNF or CNF deployed on
+it to operate as expected. The purpose of this chapter is to outline all the
```

Comment:
"operate as expected" can't be guaranteed just because the platform is conformant or the xNF is conformant. It can be only "deployed" (onboarded) as expected

Reply:
> "operate as expected" can't be guaranteed just because the platform is conformant or the xNF is conformant. It can be only "deployed" (onboarded) as expected

Fixed in [8eeb6e9](https://github.com/cntt-n/CNTT/pull/3217/commits/8eeb6e9c166cc3a4166238034dc17583f2e5fbac)